### PR TITLE
Expand upgrading Che chapter.

### PR DESCRIPTION
**Pull request by yhontyk**
> Read our [Contribution guide](https://github.com/eclipse/che-docs/blob/master/CONTRIBUTING.adoc) before submitting a PR.

### PR Checklist

As the author of this Pull Request I made sure that:

- [x] `vale` has been run successfully against the PR branch
- [x] Link checker has been run successfully against the PR branch
- [ ] Documentation describes a scenario that is already covered by QE tests, otherwise an issue has been created and acknowledged by Che QE team
- [x] Changed article references are updated where they are used (or a redirect has been set up on the docs side):
    - [x] Dashboard [branding.json](https://github.com/eclipse/che-dashboard/blob/master/src/components/branding/branding.json)
    - [x] Chectl [constants.ts](https://github.com/che-incubator/chectl/blob/master/src/constants.ts)

### CODE REVIEW

**Comment by yhontyk** [comment]
@nickboldt @davidfestal 
i've expanded the Upgrading chapter a little bit, PTAL. It is very much based on the respective [OCP docs](https://docs.openshift.com/container-platform/4.6/operators/admin/olm-upgrading-operators.html). Let me know if there's any Che-speicifc info that should be mentioned here.

**Review by MichalMaler — APPROVED** [review]

File: modules/installation-guide/partials/assembly_upgrading-che-using-operatorhub.adoc [hunk 1]
```diff
@@ -0,0 +1,20 @@
+// Module included in the following assemblies:
+//
+// upgrading-{prod-id-short}
+
+[id="upgrading-{prod-id-short}-using-operatorhub_{context}"]
+= Upgrading {prod-short} using OperatorHub
+
+This section describes how to upgrade from an earlier minor version using the Operator from OperatorHub in the OpenShift web console.
+
```
Comment (themr0c) [inline modules/installation-guide/partials/assembly_upgrading-che-using-operatorhub.adoc:9]: Missing a '.Procedure' heading

File: modules/installation-guide/partials/assembly_upgrading-che-using-operatorhub.adoc [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+OperatorHub supports `Automatic` and `Manual` upgrade strategies. 
+If the approval strategy in the subscription is set to `Automatic`, the upgrade process starts when a new version of the Operator is published.
+If the approval strategy in the subscription is set to `Manual`, the update must be manually approved every time the new version of the Operator is published.
```
Comment (themr0c) [inline modules/installation-guide/partials/assembly_upgrading-che-using-operatorhub.adoc]: Maybe a table here would increase readability?

File: modules/installation-guide/partials/proc_specifying-the-approval-strategy-of-che-in-operatorhub.adoc [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+. Click *{prod}* in the list of the installed Operators.
+
+. Navigate to the *Subscription* tab and specify the approval strategy:
++
+* *Approval*: `Automatic` 
++
+or  
++
+* *Approval*: `Manual`
```
Comment (themr0c) [inline modules/installation-guide/partials/proc_specifying-the-approval-strategy-of-che-in-operatorhub.adoc]: Maybe a table here would increase readability?

File: modules/installation-guide/partials/assembly_upgrading-che-using-operatorhub.adoc (hunk 2 above)
Comment (ibuziuk) [reply modules/installation-guide/partials/assembly_upgrading-che-using-operatorhub.adoc]: Could we clarify how the manual approval happens ? screenshot could help

**Review by ibuziuk — APPROVED** [review]
LGTM

**Pull request merged**